tests/test_carbon.py

```diff
@@ -463,6 +430,21 @@ def test_missing_keys_valuation(self):
              'price_per_metric_ton_of_c',
              'discount_rate',
              'rate_change',
-             'lulc_cur_year',
-             'lulc_fut_year'])
+             'lulc_bas_year',
+             'lulc_alt_year'])
         self.assertEqual(invalid_keys, expected_missing_keys)
+
+    def test_invalid_lulc_years(self):
+        """Test that providing an Alternate LULC year < Baseline LULC error
+        raises a ValueError"""

```

Comment:
```suggestion
        """Test Alternate LULC year < Baseline LULC year raises a ValueError."""
```
I think this get's us to less than 80 characters and the one-liner docstring sentence!